tests/network-tests/src/fixtures/membership/utils.ts

```diff
@@ -13,10 +13,14 @@ type MemberCreationParams = {
   avatarUri?: string | null
   externalResources?: MembershipMetadata.IExternalResource[] | null
   metadata: Bytes
+  is_founding_member: boolean
 }
 
 // Common code for Membership fixtures
-export function generateParamsFromAccountId(accountId: string): MemberCreationParams {
+export function generateParamsFromAccountId(
+  accountId: string,
+  isFoundingMember: boolean = false // eslint-disable-line @typescript-eslint/no-inferrable-types
```

Comment:
Is there really a need to disable this rule?
```suggestion
  isFoundingMember = false
```